charts/sonarqube/README.md

```diff
@@ -240,29 +239,29 @@ The following table lists the configurable parameters of the SonarQube chart and
-| `noProxy`               | No proxy for downloading JMX agent and install plugins, will superseed initContainer specific no proxy variables                        | ``                 |
-| `ingress-nginx.enabled` | Install Nginx Ingress Helm                                                                                                              | `false`            |
+| Parameter               | Description                                                                                                           | Default            |
+| ----------------------- | --------------------------------------------------------------------------------------------------------------------- | ------------------ |
+| `deploymentType`        | (DEPRECATED) Deployment Type (supported values are `StatefulSet` or `Deployment`)                                     | `StatefulSet`      |
+| `replicaCount`          | Number of replicas deployed (supported values are 0 and 1)                                                            | `1`                |
+| `deploymentStrategy`    | Deployment strategy. Setting the strategy type is deprecated and it will be hardcoded to `Recreate`                   | `{type: Recreate}` |
+| `priorityClassName`     | Schedule pods on priority (e.g. `high-priority`)                                                                      | `None`             |
+| `schedulerName`         | Kubernetes scheduler name                                                                                             | `None`             |
+| `affinity`              | Node / Pod affinities                                                                                                 | `{}`               |
+| `tolerations`           | List of node taints to tolerate                                                                                       | `[]`               |
+| `nodeSelector`          | Node labels for pod assignment                                                                                        | `{}`               |
+| `hostAliases`           | Aliases for IPs in /etc/hosts                                                                                         | `[]`               |
+| `podLabels`             | Map of labels to add to the pods                                                                                      | `{}`               |
+| `env`                   | Environment variables to attach to the pods                                                                           | `{}`               |
+| `annotations`           | SonarQube Pod annotations                                                                                             | `{}`               |
+| `edition`               | SonarQube Edition to use (`developer` or `enterprise`).                                                               | `None`             |
```

Comment:
Personal opinion (Feel free to ignore my comment if you disagree). Shall we add "If you want to use SonarQube Community Build, set 'community.enabled=true' instead."" here? It might be less disruptive of people using community understand from the parameter description what to use.